pykern/pkcli/test.py

```diff
@@ -18,8 +18,9 @@
 
 SUITE_D = "tests"
 _COROUTINE_NEVER_AWAITED = re.compile("RuntimeWarning: coroutine .+ was never awaited")
+_FAILED_ON_WARNINGS = "\nFAILED due to:\n"
+_PASSED_LOG_PATTERNS = (r"=+ (\d+) passed.*=+", r"PASSED")
```

Comment:
these should be re.compile'd. I think you can leave PASSED, because you don't actually know which test passed. The test will output failed.

The first regex doesn't need parens around `\d+`. I don't understand the `=+`.

Reply:
the "+=" was to match the "====== <number of passed> passed ====="